chain/vm/fvm.go

```diff
@@ -257,7 +257,6 @@ type FVM struct {
 }
 
 func NewFVM(ctx context.Context, opts *VMOpts) (*FVM, error) {
-	log.Info("using the FVM, this is experimental!")
```

Comment:
Let's leave this log in there for now, if only for us to know which VM is being run (we will def take it out before the final release).

Reply:
Merging for now, can add it back later.

Reply:
it's spamming a lot!